src/Equinor.ProCoSys.IPO.Query/GetInvitationsQueries/GetInvitations/GetInvitationsQueryHandler.cs

```diff
@@ -56,21 +60,39 @@ public async Task<Result<InvitationsResult>> Handle(GetInvitationsQuery request,
             var invitationIds = invitationForQueryDtos.Select(i => i.Id).ToList();
 
             var invitationsWithIncludes = await GetInvitationsWithIncludesAsync(_context, invitationIds, cancellationToken);
+            var codes = invitationsWithIncludes.SelectMany(x => x.Participants).Where(y => y.FunctionalRoleCode != null).Select(y => y.FunctionalRoleCode).Distinct().ToList();
```

Comment:
"var codes" - I guess we are talking about functionalRoleCodes?
If so, I would be much obliged if the variable name is renamed to functionalRoleCodes to tax my brain less :)